### PR TITLE
handbook: Create a separate page for Frontend Platform team

### PR DESCRIPTION
**Pull request by pdubroy**
Also edit Web org and Eng org pages to reflect the new structure

### CODE REVIEW

**Comment by pdubroy** [comment]
@felixfbecker - Please take a look as I also made some minor changes to the way Code Insights is organized/linked.

File: handbook/engineering/frontend-platform/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<img src="./logo.svg" alt="Sourcegraph Web Team Logo" style="width: 35%; float: right; margin-left: 1rem">
```
Comment (felixfbecker) [inline handbook/engineering/frontend-platform/index.md:1]: How about we move the `frontend-platform` (and code insights folder) to a subfolder of the `web` folder, mirroring the org structure?

File: handbook/engineering/web/index.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
     - Jean acting manager
         - [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
         - [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
         - [Murat Sutunc](../../../company/team/index.md#murat-sutunc-he-him)
-        - FQ2 frontend engineer
-        - FQ4 frontend engineer
+- [Code Insights](../frontend-platform/index.md)
+   - [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) ([Product Manager](../../product/roles/index.md#product-manager))
+   - [Felix Becker](../../../company/team/index.md#felix-becker) ([Engineering Manager](../roles.md#engineering-manager)) {#code-insights-eng}
+   - V.K. (March 2021)
```
Comment (felixfbecker) [inline handbook/engineering/web/index.md:34]:
```suggestion
   - V. K. starting 2021-03-22
```

**Review by felixfbecker — APPROVED** [review]
LGTM!

File: handbook/engineering/frontend-platform/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+<!-- Due to the markdown renderer that we use, the indentation here is sensitive. If you want to change the indentation, check that it renders correctly locally with `make serve` -->
+- [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her) ([Product Designer](../../product/roles/index.md#product-designer))
+- [Patrick Dubroy](../../../company/team/index.md#tom-ross-he-him) ([Engineering Manager](../engineering/roles#engineering-manager)) {#frontend-platform-eng}
+    - [Tom Ross](../../../company/team/index.md#tom-ross-he-him)
+    - [Felipe Janer](../../../company/team/index.md#felipe-janer-he-him)
+    - [Valery Bugakov](../../../company/team/index.md#valery-bugakov-he-him)
+
+## Contact
+
+- [#frontend-platform-chat](https://app.slack.com/client/T02FSM7DL channel or @frontend-platform in Slack.
```
Comment (umpox) [inline handbook/engineering/frontend-platform/index.md:20]:
```suggestion
- [#frontend-platform-chat](https://app.slack.com/client/T02FSM7DL) channel or @frontend-platform in Slack.
```

hey you dropped this :D

File: handbook/engineering/frontend-platform/index.md (hunk 1 above)
Comment (pdubroy) [reply handbook/engineering/frontend-platform/index.md:1]: Good idea, will do. I considered it but decided to follow the pattern that Code Insights had already set :-)

Comment (pdubroy) [reply handbook/engineering/frontend-platform/index.md]: Done.

**Pull request merged**